pkgs/development/python-modules/sqlalchemy-adapter/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3,
+  fetchPypi,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "sqlalchemy-adapter";
+  version = "1.3.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "sqlalchemy_adapter";
+    inherit version;
+    hash = "sha256-5Ze10tVKRWvc+KPHWGsjjAKqVoi7PnRoKAw3zJXme+U=";
+  };
+
+  build-system = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+  ];
+
+  dependencies = with python3.pkgs; [
+    casbin
+    sqlalchemy
+  ];
+
```

```suggestion

nativeCheckInputs = [ unittestCheckHook ];

```

This may not work. You may have to run it manually:

```

checkPhase = ''
  runHook preCheck
  python3 tests/test_adapter.py
  runHook postCheck
'';

```